worker/test/src/RTC/RTCP/TestXr.cpp

```diff
@@ -133,7 +130,7 @@ SCENARIO("RTCP XrDelaySinceLastRt parsing", "[parser][rtcp][xr-dlrr]")
 	SECTION("create RRT")
 	{
 		// Create local report and check content.
-		auto* report1 = new ReceiverReferenceTime();
+		std::unique_ptr<ReceiverReferenceTime> report1(new ReceiverReferenceTime());
```

Comment:
You can avoid using `new`.
https://stackoverflow.com/a/22571331
```suggestion
		auto report1 = std::make_unique<ReceiverReferenceTime>();
```

Reply:
good point, done.